hawc/apps/study/filterset.py

```diff
@@ -1,49 +1,45 @@
 import django_filters as df
 from django.db.models import Q
 
-from ..common.filterset import BaseFilterSet, FilterForm
+from ..common.filterset import BaseFilterSet, InlineFilterForm
 from ..myuser.models import HAWCUser
 from . import constants, models
 
 
 class StudyFilterSet(BaseFilterSet):
-    citation = df.CharFilter(
-        method="filter_citation", label="Citation", help_text="Authors, year, title, etc."
-    )
-    identifier = df.CharFilter(
-        field_name="identifiers__unique_id",
-        lookup_expr="icontains",
-        label="Identifier",
-        help_text="Database identifier<br/>(PubMed ID, DOI, HERO ID, etc)",
+    search = df.CharFilter(
+        method="filter_search",
+        label="Citation/Identifier",
+        help_text="Filter by citation (authors, year, title, etc) or datatbase identifier (PubMed ID, DOI, HERO ID, etc.)",
```

Comment:
I like these combo fields!
Again, try to name them more descriptively (maybe "citation_or_id").
You've got a typo in the help text here (datatbase), but I'd get rid of that word entirely (Filter by citation (...) or identifier (...).